sale_packaging_price/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/lgpl
+    :alt: License: LGPL-3
+
+====================
+Sale Packaging Price
+====================
+
+This module extends the functionality of sale workflow and allows you to assign
+a selling price of a complete package. This price is split for having the price
+unit according the number of pieces that fit in that packaging, so there can be
+rounding issues to get the exact price that are warned if happens. It also adds
+a field to set the package material.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Inventory > Configuration > Settings*.
+#. Enable *Traceability > Packages > Record packages used on packing: pallets,
+   boxes, ...*
+#. Enable *Products > Packaging Methods > Manage available packaging options
+   per products*.
+#. *Apply*.
+
+Usage
+=====
+
+First, ensure you have permissions for *Technical Settings / Manage Product
```

Comment:
I can't find this permission in Administrator user form and access rights tab:

![image](https://cloud.githubusercontent.com/assets/1881111/17440841/a3c503ae-5b2e-11e6-8c69-26af4e81b858.png)